code/modules/power/apc.dm

```diff
@@ -1548,3 +1548,20 @@ var/zapLimiter = 0
 /obj/machinery/power/apc/powered()
 	//Always powered
 	return 1
+
+/obj/machinery/power/apc/proc/is_not_default()
+	return operating != 1 || chargemode != 1 || (!shorted && (equipment != 3 || lighting != 3 || environ != 3)) || coverlocked != 1
```

Comment:
if `operating`, `chargemode` and others can only be 1/0, turn them to TRUE/FALSE